src/commands/kv/mod.rs

```diff
@@ -138,6 +141,41 @@ fn help(error_code: u16) -> &'static str {
     }
 }
 
+pub fn validate_target(target: &Target) -> Result<(), failure::Error> {
+    let mut missing_fields = Vec::new();
+
+    if target.account_id.is_empty() {
+        missing_fields.push("account_id")
+    };
+    if target.name.is_empty() {
+        missing_fields.push("name")
```

Comment:
idk if this makes sense here.

Reply:
this makes sense. Happy to remove this!

Reply:
oh wait, isn't `name` required when you create a KV namespace?